.env

```diff
@@ -35,6 +46,9 @@ export CT_USE_REAL_API_DATA=false
 # HSES_API_PASSWORD is the Basic Auth password to access the HSES API
 # export HSES_API_PASSWORD=
 
+# TTA_HUB_API_HOSTNAME is the host to access the TTA Hub API
+# export TTA_HUB_API_HOSTNAME=
```

Comment:
Development defaults to hitting the tta hub staging server, so this only needs to be set to override that.

It's a production ENV var because it'll have to switch from stage to prod